backend/api/Database/Models/Robot.cs

```diff
@@ -28,6 +28,8 @@ public class Robot
 
         public string CurrentAsset { get; set; }
 
+        public string CurrentAssetDeckId { get; set; }
+
```

Comment:
Is it perhaps better to use
```suggestion
        public AssetDeck CurrentAssetDeckId { get; set; }
```

Reply:
Do we want to remove CurrentAsset as well and replace it with CurrentAssetDeckId ?

Reply:
We chouldn't change currentAsset but "string CurrentAssetDeckId" should be switched out with "AssetDeck CurrentAssetDeck"

Reply:
Ideally we replace both `string CurrentAsset` and `string CurrentAssetDeckId` with `AssetDeck CurrentAssetDeck`

If you get some issues with this, I can probably help =)